oojs.js

```diff
@@ -10,3 +10,27 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist {
+  constructor (title, songs) {
+    this.title = title;
+    this.songs = ["Workout", "Fun", "Zumba"];
+    this.favorites = 0;
+  }
+  addSong(songs) {
+    this.songs.push(songs);
+    console.log(this.songs);
+  }
+}
+
+const gym = new Playlist ("Gym", "Run")
```

Comment:
Remember that `songs` is initialized as an array, so you want to make sure you pass in an array as an argument.